### PR TITLE
 [86endm2jf] Team Page

### PR DESCRIPTION
**Pull request by cl7-ab2**
create team page

### CODE REVIEW

**Comment by pathusutariya** [comment]
Wrong title.  It's not as per I told you so.
Use Following title.
[86endm2jf] Team Page.

File: team_page.html
Comment (pathusutariya) [inline team_page.html:1]: Bootstrap is missing.

File: team_page.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<table>
+	<tr>
+		<th>Name</th>
+		<th>Job title</th>
+		<th>Email address</th>
+
+	</tr>
+	<tr>
+		<td>ashish</td>
+		<td>php developer</td>
```
Comment (pathusutariya) [inline team_page.html]:
```suggestion
		<td>PHP Developer</td>
```

File: team_page.html
Comment (pathusutariya) [inline team_page.html:1]: Name should be capitalized. please check proper typo.

**Review by pathusutariya — CHANGES_REQUESTED** [review]
Error commented.

**Review by cl7-ab2 — COMMENTED** [review]
boostrap added
name all capitalize
and title change

File: team_page.html
Comment (pathusutariya) [reply team_page.html:1]: `d` is not capital in developer

**Review by pathusutariya — APPROVED** [review]
All done.

**Pull request merged**